app/api/event_invoices.py

```diff
@@ -120,13 +131,13 @@ def before_create_object(self, data, view_kwargs):
             data['discount_code_id'] = discount_code.id
 
     view_kwargs = True
-    decorators = (is_admin,)
```

Comment:
Should event-invoices be visible to everyone? I think only organiser. @niranjan94 ?